### PR TITLE
Angular2 Quickstart - edits to all docs, and includes

### PR DESCRIPTION
**Pull request by Catografix**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by Catografix** [comment]
@chenkie Please review

File: articles/client-platforms/_includes/_rules-introduction.md [hunk 1]
```diff
@@ -1 +1 @@
-Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](/rules).
\ No newline at end of file
+Rules are one of the most powerful features of Auth0. As JavaScript functions that execute each time a user authenticates, Rules serve as middleware to extend the flexibility of your authentication flow. For a detailed description of Rules, see the [full documentation](/rules).
```
Comment (chenkie) [inline articles/client-platforms/_includes/_rules-introduction.md]: I think we should lowercase `rules` when not at the start of a sentence here.

File: articles/client-platforms/angular2/07-authorization.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ public isAdmin() {
 ...
 ```
 
-We are also storing the target url to make a redirect when successfully login, so let's add that:
+After successfully login, redirect to the saved URL:
```
Comment (chenkie) [inline articles/client-platforms/angular2/07-authorization.md]: After successfully login => After logging in successfully

File: articles/client-platforms/angular2/08-calling-apis.md [hunk 3]
```diff
@@ -7,13 +7,13 @@ description: This tutorial will show you how to use angular2-jwt library in Angu
   link: 'https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample/tree/master/08-Calling-Api',
 }) %>_
 
-The reason for implementing authentication in the first place is to protect information. In this case your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. Auth0 suggests you conform to RFC standard by sending the token through Authorization header.
+Auth0 exposes an assortment of API endpoints to assist you with authentication in your application. Auth0 suggests you conform to the RFC standard by sending the token through Authorization header when calling an API.
 
 ## Meet AuthHttp
```
Comment (chenkie) [inline articles/client-platforms/angular2/08-calling-apis.md]: We should change this heading. Let's call it `Sending Authenticated HTTP Requests`

File: articles/client-platforms/angular2/08-calling-apis.md [hunk 4]
```diff
@@ -7,13 +7,13 @@ description: This tutorial will show you how to use angular2-jwt library in Angu
...
 
-In order to make an authorized request, [angular2-jwt](https://github.com/auth0/angular2-jwt) provides the `AuthHttp` helper which has the same `Http` module interface but automatically add the authorization header to the requests.
+In order to make an authorized request, [angular2-jwt](https://github.com/auth0/angular2-jwt) provides the `AuthHttp` helper which has the same interface as the `Http` module but automatically adds the authorization header to requests.
```
Comment (chenkie) [inline articles/client-platforms/angular2/08-calling-apis.md]: authorized request => authenticated request

File: articles/client-platforms/_includes/_rules-introduction.md (hunk 1 above)
Comment (Catografix) [inline articles/client-platforms/_includes/_rules-introduction.md]: Yes, I've been indecisive about this, but when referring to Rules as an Auth0 feature, I tend to like capitalizing it. But either way ok with me.

**Comment by Catografix** [comment]
Waiting to have the link redirects resolved before final edit.

File: articles/client-platforms/angular2/00-intro.md [hunk 5]
```diff
@@ -5,46 +5,17 @@ description: A simple, multi-step quickstart guide to setup and manage authentic
...
 This simple, multi-step quickstart will guide you through setting up and managing authentication in your Angular2 JS apps using Auth0.  
 
-## Seed & Samples
-
-There are two options for following along with this quickstart:
-
-* Download the [seed project](https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample/tree/master/00-Starter-Seed). The seed project is a regular Angular2 app (based on [https://github.com/angular/quickstart](https://github.com/angular/quickstart)) containing only the Auth0 dependencies in the HTML that can serve as a starting point for this quickstart.
-
-* Download the [samples](https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample) that contain the examples for each step.
+<%= include('../../_includes/_github', {
```
Comment (Catografix) [inline articles/client-platforms/angular2/00-intro.md:8]: @chenkie Are these changes correct? And the link to the starter seed OK?

**Comment by Catografix** [comment]
Preview OK: https://auth0-docs-content-pr-1931.herokuapp.com/docs/quickstart/spa/angular2

**Comment by Catografix** [comment]
I accidentally included the new `redirect-urls.json` from #1933 which isnt merged yet.
How can I un-include this file, or is it OK as is?

**Pull request merged**